Logging/LogHandler/FileLogHandler.cs

```diff
@@ -10,12 +9,30 @@ namespace Anvil.CSharp.Logging
     /// </summary>
     public class FileLogHandler : AbstractAnvilBase, ILogHandler
     {
+        /// <summary>
+        /// The type of log files the handler should generate.
+        /// </summary>
+        public enum LogType
+        {
+            /// <summary>
+            /// New logs are appended to the existing log file.
+            /// </summary>
+            Append,
+            /// <summary>
+            /// A new log file is created each session, overwriting the previous one if it exists.
```

Comment:
```suggestion
            /// A new log file is created, overwriting the previous one if it exists.
```

Super minor, but we should avoid mention of the session as the app layer will handle that